service/controller/v13/cluster_resource_set.go

```diff
@@ -110,15 +114,26 @@ func NewClusterResourceSet(config ClusterResourceSetConfig) (*controller.Resourc
 		return nil, microerror.Maskf(invalidConfigError, "%T.APIWhitelist.SubnetList must not be empty when %T.APIWhitelist is enabled", config)
 	}
 
-	var encrypter *kms.Encrypter
-	{
+	var encrypterBackend encrypter.Interface
+	if config.Encrypter == "vault" {
```

Comment:
I'd prefer a constant for the string here.

Reply:
Good point, added to the encrypter package.